pkg/controller/podautoscaler/horizontal.go

```diff
@@ -329,6 +334,12 @@ func (a *HorizontalController) reconcileAutoscaler(hpa *autoscaling.HorizontalPo
 		if desiredReplicas > hpa.Spec.MaxReplicas {
 			desiredReplicas = hpa.Spec.MaxReplicas
 		}
+
+		// Do not upscale too much to prevent incorrect rapid increase of the number of master replicas caused by
+		// bogus CPU usage report from heapster/kubelet.
```

Comment:
can you add a bug number here so that future readers of this recognize what we're talking about here (I'm assuming this is to avoid the persistent issues with overflow/astronomically high CPU)?